src/feed.tsx

```diff
@@ -8,18 +10,79 @@ export interface Properties {
   app: string;
 }
 
-export class Feed extends React.Component<Properties> {
+export interface State {
+  feed: FeedItemModel[];
+  hasMore: boolean;
+  pageNumber: number;
+}
+
+export class Feed extends React.Component<Properties, State> {
+  static pageSize: number = 5;
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      feed: this.props.items.slice(0, Feed.pageSize) || [],
+      hasMore: false,
+      pageNumber: 2,
```

Comment:
are we starting at 2 to get around the negative slice? i *think* it feels better to just handle the negative in place, if that's the case. (eg. .. `Math.max(pageNumber - 1, 0)`) but we could also just start with pageNumber of 1, and it shouldn't be an issue? i think either 0 or 1 based page numbers make sense, but starting at 2 feels like it will be confusing.